src/ol_infrastructure/applications/concourse/__main__.py

```diff
@@ -0,0 +1,532 @@
+        )
+    ],
+    egress=default_egress_args,
+    vpc_id=ops_vpc_id,
+)
+
+# Create security group for Concourse Postgres database
+concourse_db_security_group = ec2.SecurityGroup(
+    f"concourse-db-access-{stack_info.env_suffix}",
+    name=f"concourse-db-access-{stack_info.env_suffix}",
+    description="Access from Concourse instances to the associated Postgres database",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[concourse_web_security_group.id],
+            # TODO: Create Vault security group to act as source of allowed
+            # traffic. (TMM 2021-05-04)
+            cidr_blocks=[operations_vpc["cidr"]],
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            description="Access to Postgres from Concourse web nodes",
+        ),
+    ],
+    tags=aws_config.tags,
+    vpc_id=ops_vpc_id,
+)
+
+
+##########################
+#     Database Setup     #
+##########################
+concourse_db_config = OLPostgresDBConfig(
+    instance_name=f"concourse-db-{stack_info.env_suffix}",
+    password=concourse_config.require("db_password"),
+    subnet_group_name=operations_vpc["rds_subnet"],
+    security_groups=[concourse_db_security_group],
+    tags=aws_config.tags,
+    db_name="concourse",
+    **defaults(stack_info)["rds"],
+)
+concourse_db = OLAmazonDB(concourse_db_config)
+
+concourse_db_vault_backend_config = OLVaultPostgresDatabaseConfig(
+    db_name=concourse_db_config.db_name,
+    mount_point=f"{concourse_db_config.engine}-concourse",
+    db_admin_username=concourse_db_config.username,
+    db_admin_password=concourse_config.require("db_password"),
+    db_host=concourse_db.db_instance.address,
+)
+concourse_db_vault_backend = OLVaultDatabaseBackend(concourse_db_vault_backend_config)
+
+if stack_info.env_suffix == "production":
+    consul_datacenter = "operations"
+else:
+    consul_datacenter = "operations-qa"
+concourse_db_consul_node = Node(
+    "concourse-instance-db-node",
+    name="concourse-postgres-db",
+    address=concourse_db.db_instance.address,
+    datacenter=consul_datacenter,
+)
+
+concourse_db_consul_service = Service(
+    "concourse-instance-db-service",
+    node=concourse_db_consul_node.name,
+    name="concourse-postgres",
+    port=concourse_db_config.port,
+    meta={
+        "external-node": True,
+        "external-probe": True,
+    },
+    checks=[
+        ServiceCheckArgs(
+            check_id="concourse-instance-db",
+            interval="10s",
+            name="concourse-instance-id",
+            timeout="60s",
+            status="passing",
+            tcp=f"{concourse_db.db_instance.address}:{concourse_db_config.port}",  # noqa: WPS237,E501
+        )
+    ],
+)
+
+##########################
+#     EC2 Deployment     #
+##########################
+
+# Create load balancer for Concourse web nodes
+web_lb = lb.LoadBalancer(
```

Comment:
Should this be named `concourse_web_lb`?

Reply:
It's a local variable so it's not that important

Reply:
But yes, probably